src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="container__calendar calendar calendar--month-length--28 calendar--start-day--wed">
```

Comment:
```suggestion
      <div class="calendar calendar--month-length--28 calendar--start-day--wed">
```

Why do you need container modifier?